apps/web/src/styles/7-components/_table.scss

```diff
@@ -1,44 +1,47 @@
 .govuk-accordion__section-content {
-  .govuk-summary-list:last-child {
-    margin-bottom: 25px;
-  }
-  .govuk-table__row:last-child {
-    .govuk-table__header, .govuk-table__cell {
-      border-bottom: none;
-    }
-  }
+	.govuk-summary-list:last-child {
+		margin-bottom: 25px;
+	}
+
+	.govuk-table__row:last-child {
+		.govuk-table__header, .govuk-table__cell {
+			border-bottom: none;
+		}
+	}
 }
 
 .pins-table {
-  &--fixed {
-    table-layout: fixed;
-  }
-  &__cell {
-    &--xs {
-      width: 80px;
-    }
-    &--s {
-      width: 130px;
-    }
-    &--m {
-      width: 180px;
-    }
-    &--l {
-      width: 270px;
-    }
-    &--xl {
-      width: 330px;
-    }
-  }
+	&--fixed {
+		table-layout: fixed;
+	}
+
+	&__cell {
+		&--xs {
+			width: 80px;
+		}
+		&--s {
+			width: 140px;
```

Comment:
Changed this to 140 from 130 to fix the wrapping.

<img width="1034" alt="image" src="https://user-images.githubusercontent.com/46779/166436404-982b3908-1716-409d-baf9-0408a8763b0e.png">